README.md

```diff
@@ -54,7 +54,133 @@ hasp_lvgl:
               message: Hello {{ text }}
 ```
 
-in the event service call you can use any variable coming from the MQTT message besides the actual event
+In the event service call any variable coming from the MQTT message can be used between curly brackets. 
+
+### Example 1: cover control with state feedback
+![kép](https://user-images.githubusercontent.com/1550668/112142921-77daf580-8bd7-11eb-9626-ebfb3423629d.png)
+<br>UI theme set to `Hasp Light` in plate's web interface.
+
```

Comment:
we also need to tell users which HA entities are involved

Reply:
can be seen from the code, its self-explanatory

Reply:
you don't deal with end user too often, now do you 😄 ?

I also think it's better to name our entities in English (I'm also to blame in my example)

Reply:
I see what you mean, I already translated most of them to cover_1 and 2 but only one was left from my code, sorry.
I am also having a hard time reading others code but that makes me learn and understand better.